src/renderer/components/attachment/menuAttachment.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import {
+  Button,
+  Position,
+  Popover,
+  Menu,
+  MenuItem,
+  IconName,
+} from '@blueprintjs/core'
+import { useTranslationFunction } from '../../contexts'
+import { runtime } from '../../runtime'
+
+//function to populate Menu
+const MenuAttachmentItems = ({
+  itemsArray,
+}: {
+  itemsArray: MenuAttachmentItemObject[]
+}) => {
+  const tx = useTranslationFunction()
+  return (
+    <>
+      {itemsArray.map((item: MenuAttachmentItemObject) => (
+        <MenuItem
+          key={item.id}
+          icon={item.icon}
+          text={item.text}
+          onClick={item.onClick}
+          aria-label={tx(item.attachment)}
+        />
+      ))}
+    </>
+  )
+}
+//main component that creates the menu and popover
+const MenuAttachment = ({
+  addFileToDraft,
+}: {
+  addFileToDraft: (file: string) => void
+}) => {
+  const addFilenameFile = async () => {
+    //function for files
+    const file = await runtime.showOpenFileDialog({
+      filters: [
+        {
+          name: 'Files',
+          extensions: [
+            'doc',
+            'docx',
+            'xls',
+            'xlsx',
+            'ppt',
+            'ppt',
+            'pdf',
+            'txt',
+            'csv',
+            'log',
+            'zip',
+          ],
+        },
+      ],
+      properties: ['openFile'],
+      defaultPath: runtime.getAppPath('home'),
+    })
+    if (file) {
+      addFileToDraft(file)
+    }
+  }
+  //function for media
+  const addFilenameMedia = async () => {
+    const file = await runtime.showOpenFileDialog({
+      filters: [
+        {
+          name: 'Media',
+          extensions: ['jpg', 'jpeg', 'png', 'gif', 'mkv', 'avi', 'mp4'],
```

Comment:
Let's put this array into a shared constant variable. This way we can use it at various places and don't have to copy it. I would propose having it as `const MEDIA_EXTENSIONS = ['jpg'..., 'mp4'] in `src/shared/constants.ts`